devito/finite_differences/coefficients.py

```diff
@@ -224,6 +224,33 @@ def generate_subs(i):
         rules = {}
         for i in self.coefficients:
             rules.update(generate_subs(i))
+        return rules
+
+    def update_rules(self, obj):
+        """Update the specified rules to reflect staggering in an equation"""
+        # Determine which 'rules' are expected
+        sym = get_sym(self._function_list)
```

Comment:
Sorry, which symbol is this getting again? Do functions with symbolic coefficients not already have a property for this?